src/User32.Desktop/User32.Helpers.cs

```diff
@@ -47,5 +47,24 @@ public static unsafe string GetClassName(IntPtr hWnd)
 
             return new string(className, 0, count);
         }
+
+        /// <summary>
+        /// Retrieves from the clipboard the name of the specified registered format.
+        /// </summary>
+        /// <param name = "format">The type of format to be retrieved. This parameter must not specify any of the predefined clipboard formats.</param>
+        /// <returns>The format name string.</returns>
+        /// <exception cref="Win32Exception">Thrown when an error occurs.</exception>
+        public static unsafe string GetClipboardFormatName(int format)
+        {
+            const int bufferSize = 256;
```

Comment:
> I believe you meant "there is no way".

Yes :)

I was more commenting on the windows API design here that on the specific code, lot of samples on GitHub seem to call it with 255/256/MAX_PATH variants (Others are using 100, 1000 or 1024) so we aren't really departing from what's already done elsewhere.

And I fear that API consumers won't have better information about the potential size than us so it's just moving the uncertainty one level up.